securedrop/manage.py

```diff
@@ -210,6 +242,48 @@ def listdir_fullpath(d):
     return 0
 
 
+def translate(args):
+    messages_file = os.path.join(args.translations_dir, 'messages.pot')
+
+    if args.extract_update:
+        sh("""
+        set -xe
+
+        mkdir -p {translations_dir}
+
+        pybabel extract \
+        --charset=utf-8 \
+        --mapping={mapping} \
+        --output={messages_file} \
+        --project=SecureDrop \
+        --version={version} \
+        --msgid-bugs-address='securedrop@freedom.press' \
+        --copyright-holder='Freedom of the Press Foundation' \
+        {sources}
+
+        sed -i '/^#, fuzzy$/d' {messages_file}
+
+        test $(ls {translations_dir} | wc -l) = 1 && exit 0
+
+        pybabel update \
+        --input-file {messages_file} \
+        --output-dir {translations_dir} \
+        --no-fuzzy-matching --ignore-obsolete
+        """.format(translations_dir=args.translations_dir,
+                   mapping=args.mapping,
+                   messages_file=messages_file,
+                   version=version.__version__,
```

Comment:
Perhaps we should add an option to either manually pass a `version` string or an abbreviated commit hash? The reason being, if we are preparing translations to be incorporated into a particular release, the version string here will not be updated and we may get confused about which version of the software the translation text files came from / are for.